templates/dependencies.html

```diff
@@ -0,0 +1,42 @@
+<div class="bower-config-file">
```

Comment:
I renamed all the installed packages to dependecies.
So installed-packages.html > dependencies.js